### ISSUE TITLE
Failure on all operations since 0.2.7

### ISSUE THREAD

**Issue by sdomeshok** [Mongey/terraform-provider-kafka#133]
All operations are returning
`Error: rpc error: code = Canceled desc = context canceled` for each resource. `0.2.6` did not work due to the version mismatch issue, so `0.2.5` is the most recent non-failing version.

Sample TF_LOG output:
```
2020/06/25 17:54:15 [ERROR] module.foo: eval: *terraform.EvalReadState, err: rpc error: code = Canceled desc = context canceled
2020/06/25 17:54:15 [ERROR] module.foo: eval: *terraform.EvalSequence, err: rpc error: code = Canceled desc = context canceled
```

**Comment by Mongey** [comment]
sorry you ran into this @sdomeshok ... I'm trying to get some better test coverage so that we can iron out these issues.
Can you provide your tf kafka provider settings /  some information about your kafka setup?

**Comment by sdomeshok** [comment]
Thanks @Mongey! 
3 broker cluster, dev at 2.4.1, staging and prod at 2.1.1, 0.2.7 is failing in each environment.


```
provider "kafka" {
  bootstrap_servers = var.bootstrap_servers
  skip_tls_verify   = true
  ca_cert           = local.cert_path
  tls_enabled       = true
  sasl_username     = var.kafka_username
  sasl_password     = var.kafka_password
  sasl_mechanism    = "scram-sha256"
}
```

Any other useful info I can give you? What I can say is that it is a strict regression from 0.2.5

**Comment by Mongey** [comment]
This should be resolved in the 0.2.8 / 0.2.9 release.
